web/testing/cypress/e2e/ID003_Create_Unique_Event.cy.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-undef */
+/// <reference types="cypress" />
+
+describe("Basic app navigation", () => {
+  beforeEach(() => {
+    cy.viewport(1440, 900);
+
+    cy.visit("/");
+  });
+
+  it("Create Event", () => {
+    const userEmail = `e2e+${Math.ceil(
+      Math.random() * 1000000000000000
+    )}@testdfas.com`;
+    console.log(userEmail);
+    cy.get(".w-full > .flex").click();
+    cy.origin(
+      "https://oncampus.us.auth0.com",
+      { args: { userEmail } },
+      ({ userEmail }) => {
+        cy.get("a").contains("Sign up").click();
+        cy.get("input#email").type(userEmail); //me might have to go through signup every time and create a new account? might trigger the auth0 limits if we do that...
+        cy.get("input#password").type("#e5PUJe9@mroL$", { log: false });
+        cy.contains("button[value=default]", "Continue").click();
+        cy.get("button").contains("Accept").click();
+      }
+    );
+    cy.visit("/schedule");
+    cy.get(":nth-child(4) > :nth-child(3)").dblclick();
+    cy.wait(1000)//bad practice, but we can't really fix the shedule interface...
```

Comment:
Maybe the wait time should be longer because I still get this sometimes: 

<img width="922" alt="image" src="https://user-images.githubusercontent.com/68136998/221946038-1b7f9708-8aa5-4c91-aa5a-dfb200641a32.png">